src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -180,21 +180,19 @@ public void visitToken(DetailAST ast) {
      * Checks whether token is followed by a whitespace.
      *
      * @param targetAST Ast token.
-     * @param line The line associated with the ast token.
+     * @param codePoints the array of Unicode code points
      * @return true if ast token is followed by a whitespace.
      */
-    private static boolean isFollowedByWhitespace(DetailAST targetAST, String line) {
+    private static boolean isFollowedByWhitespace(int[] codePoints, DetailAST targetAST) {
```

Comment:
Please do not change the order of parameters in existing methods, there are a few cases of this.

Reply:
Ok,